lib/sorbet-rails/active_record_rbi_formatter.rb

```diff
@@ -81,6 +81,62 @@ def generate_active_record_relation_rbi
       class_rbi.create_method('one?', return_type: "T::Boolean")
     end
 
+    parlour.root.create_class("ActiveRecord::AssociationRelation", superclass: "ActiveRecord::Relation") do |class_rbi|
+      class_rbi.create_constant(
+        "Elem",
+        value: "type_member(fixed: T.untyped)",
+      )
+
+      # Ideally we shouldn't need to define these since this class inherits from
+      # ActiveRecord::Relation but the activerecord.rbi that sorbet generates
+      # defines some methods which sorbet finds instead of the methods inherited
+      # by ActiveRecord::Relation. Some of these methods have different arity or
+      # parameters than the ones defined by `create_elem_specific_query_methods` so
+      # we need to match the signatures in that conflicting rbi.
+      build_methods = %w(new build create create!)
+      build_methods.each do |build_method|
+        class_rbi.create_method(
+          build_method,
+          parameters: [
+            Parameter.new("*args", type: "T.untyped"),
```

Comment:
I think this is wrong? It was changed to have `attributes` just like the proxy methods: https://github.com/rails/rails/commit/f1c1c43022c24e27a02d698fd29e7955c10d2f03

The sorbet-rails tests fail now if you run the tests and it installs Rails 6.0.3. (I'm not sure if 6.0.2/6.0.1 are effected?)